gammapy/datasets/map.py

```diff
@@ -2886,3 +2865,46 @@ def irf_position_changed(self):
 
         return changed
 
+
+    def norm_only_changed(self):
```

Comment:
This method needs simplification. A few ideas:
- Identify the "norm/amplitude" parameter on MapEvaluator init
- Just check the norm like:
```python3
@property
def parameters_norm_changed(self, reset=True):
    """Parameters changed"""
    value = self.model.parameters.value[self._norm_idx]
    changed = self._cached_parameter_values[sefl._norm_idx] == value

    if changed and reset:
        self._cached_parameter_values = values

    return changed

```

- In addition maybe introduce the re-norm factor like:
```python3

def renorm():
    value = self.model.parameters.value[self._norm_idx]
    value_cached = self._cached_parameter_values[self._norm_idx]
    return value / value_cached    
```

Reply:
done, not exactly like this but that's simpler now